### PR TITLE
Dtspb 2508 docker changes am

### PR DESCRIPTION
**Pull request by isha2040**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-2508

### Change description ###

am role assignment changes in docker

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -81,6 +82,12 @@ services:
       ELASTIC_SEARCH_DATA_NODES_HOSTS: "http://ccd-elasticsearch:9200"
       DATA_STORE_DEFAULT_LOG_LEVEL: INFO
       CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api:4455
+      CCD_DEFAULTPRINTURL: https://return-case-doc-ccd.nonprod.platform.hmcts.net/jurisdictions/:jid/case-types/:ctid/cases/:cid
+      ENABLE_ATTRIBUTE_BASED_ACCESS_CONTROL: "${ENABLE_ATTRIBUTE_BASED_ACCESS_CONTROL:-true}"
+      ENABLE_PSEUDO_ROLE_ASSIGNMENTS_GENERATION: "${ENABLE_PSEUDO_ROLE_ASSIGNMENTS_GENERATION:-true}"
+      ENABLE_PSEUDO_ACCESS_PROFILES_GENERATION: "${ENABLE_PSEUDO_ACCESS_PROFILES_GENERATION:-true}"
```
Comment (nigeldunne) [inline docker-compose.yml:88]: is the '-' meant to be there?

File: docker-compose.yml [hunk 2]
```diff
@@ -662,6 +672,46 @@ services:
...
+    image: hmctspublic.azurecr.io/am/role-assignment-service:latest
+    environment:
+      ROLE_ASSIGNMENT_DB_HOST: "shared-database"
+      ROLE_ASSIGNMENT_DB_NAME: role_assignment
+      ROLE_ASSIGNMENT_DB_PORT: 5432
+      ROLE_ASSIGNMENT_DB_USERNAME: "${DB_USERNAME}"
+      ROLE_ASSIGNMENT_DB_PASSWORD: "${DB_PASSWORD}"
+      ROLE_ASSIGNMENT_DB_OPTIONS:
+      AM_ROLE_ASSIGNMENT_SERVICE_SECRET: "AAAAAAAAAAAAAAAA"
+      LD_SDK_KEY: sdk-4d50eb6e-8400-4aa7-b4c5-8bdfc8b1d844
```
Comment (nigeldunne) [inline docker-compose.yml]: key

File: docker/ccd-database/init-db.sh [hunk 3]
```diff
@@ -22,3 +22,7 @@ psql -v ON_ERROR_STOP=1 --username postgres --set USERNAME=$DB_USERNAME --set PA
 EOSQL
   echo "Database $service: Created"
 done
+# should be accomplished by above additional service for role_assignment
```
Comment (nigeldunne) [inline docker/ccd-database/init-db.sh]: remove the comment

**Review by nigeldunne — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (isha2040) [reply docker-compose.yml:88]: not sure Nigel, i didnt change anything in here , same as you and Sanjay initial put together . I can try removing it doesnt look right to me either

File: docker-compose.yml (hunk 2 above)
Comment (isha2040) [reply docker-compose.yml]: thank you well spotted , will remove it and lets use the env var we set already

File: docker-compose.yml (hunk 1 above)
Comment (isha2040) [reply docker-compose.yml:88]: i found many instances of this :- in the same docker compose , i would leave it as it for now . Dont think it impacts

**Review by nigeldunne — APPROVED** [review]

**Pull request merged**